streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationActionServiceImpl.java

```diff
@@ -381,6 +383,15 @@ public void start(Application appParam, boolean auto) throws Exception {
     ApiAlertException.throwIfTrue(
         !application.isCanBeStart(), "[StreamPark] The application cannot be started repeatedly.");
 
+    if (FlinkExecutionMode.YARN_APPLICATION == application.getFlinkExecutionMode()
```

Comment:
We can improve this code：

if (FlinkExecutionMode.isYarnMode(application.getFlinkExecutionMode())) {
...
}